dames.c

```diff
@@ -313,6 +414,189 @@ void Affichage_deplacement(char tab[][COLONNE_DAMIER], char couleur,int ligne_pi
 }
 
 
+int validation_manger_pion(char tab[][COLONNE_DAMIER], char couleur,int ligne_pion_selectionner,int colonne_pion_selectionner){
+
 
 
+    char couleur_oposant;
+    int mbd;
```

Comment:
mdb, mbg, mhd, mhg ? What the hell?

...

If that's really usefull to contract all movement names you might add a comment and use it in your function names too.